tailwind.config.ts

```
@@ -28,6 +28,9 @@ const config: Config = {
       },
     },
   },
-  plugins: [],
+  variants: {
+    scrollbar: ['rounded'],
+  },
+  plugins: [require('tailwind-scrollbar')],
```

ESLint

[error] A `require()` style import is forbidden.

(@typescript-eslint/no-require-imports)